templates/web/base/report/new/councils_text_all.html

```diff
@@ -2,12 +2,19 @@
 [% DEFAULT list_of_names = default_list %]
 
 <p>
-[%
+<span class="js-hide-if-private-category">[%
```

Comment:
Here, you know the category, and whether it's public or private, don't you?

Reply:
Yes, but the contents of this template is only rendered and included in `/report/new/ajax` output on a per-category basis for unresponsive bodies/categories AFAICT. I've pushed 6faa885a0e530462c4d393f69b29982e27a116b7 which rejigs things a bit to include `councils_text` for `non_public` categories, does that seem like a better approach?

Reply:
Yeah, that looks better, doesn't it. Perhaps it would be better separate and down next to/with the existing inclusion of that template for unresponsive (general tidying of that fn, why is $generate not next to where it is used? It's performing the same purpose for the category_extras.html template) - with a note saying this template needs to be included whenever it would be changed from the default output, and that is currently unresponsive or non_public.

Reply:
Good idea. 155a00074f731ff694e02ca1cee24dc755668853 rejigs it a bit.